logicalplan/distribute.go

```diff
@@ -20,10 +19,6 @@ import (
 	"github.com/thanos-io/promql-engine/query"
 )
 
-var (
-	RewrittenExternalLabelWarning = errors.Newf("%s: rewriting an external label with label_replace could lead to unpredictable results", annotations.PromQLWarning.Error())
```

Comment:
should we keep this as info annotation instead?

Reply:
I wouldn't say that results should be unpredictable with this change, but there could be performance issues since a smaller part of the query can be distributed. Should we say that instead?

Reply:
Yeah, exactely! We could say that that might prevent valuable optimizations.